extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -572,17 +596,25 @@ export class ConsentUI {
   resetIframe_() {
     const {classList} = this.parent_;
 
-    // Remove the iframe active to go back to our normal height
-    classList.remove(consentUiClasses.iframeActive);
+    // Remove the iframe active (lightbox) to go back to our normal height
+    classList.remove(
+      this.lightboxEnabled_
+        ? consentUiClasses.iframeActiveLightbox
+        : consentUiClasses.iframeActive
+    );
+
+    // Remove border if enabled
+    if (this.enableBorder_ && !this.lightboxEnabled_) {
```

Comment:
Why aren't these styles part of the styling for the classname above?

Reply:
Borders are optional for non-modal. I will change the conditionality of this to make it more clear.